src/components/combobox/combobox.scss

```diff
@@ -166,8 +166,8 @@
 
 .list-container {
   @apply bg-foreground-1
-    max-h-screen
     overflow-y-auto;
+  max-block-size: 50vh;
```

Comment:
This could be a neat util/mixin to add.

Reply:
I'll add a screener and see if we can do a tailwind option for this.

It's still not perfect if you open slightly below half the page and it doesn't open above. We can maybe tweak it to 40%.